docs/adapt-theme-to-brand.md

```diff
@@ -3,4 +3,89 @@ id: adapt-theme-to-brand
 title: Adapt the theme to your brand
 ---
 
-Adapt the theme to your brand
\ No newline at end of file
+When you start a new project, one of the first things that you will need to do
+is to adapt the look and feel of Front Commerce to match your brand's style.
+
+Obviously, it is not something that can be done in just a few minutes because
+your identity shines in the details. However, you can do all the heavylifting
+pretty quickly by customizing what is called **Design Tokens**.
```

Comment:
I think it would be a good idea to emphasizes here on the fact that is could be a good starting point when prototyping with FCL.